### PR TITLE
Removed duplicate block tests and renamed tests.

### PR DESCRIPTION
**Pull request by moniika**
## The basics

- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

#2870

### Proposed Changes
Removed jsunit tests for blocks as they were already covered in mocha tests. Additionally, renamed unit tests to be more clear in test results.

### Reason for Changes
Explained in linked issue.

### Test Coverage
Ran mocha tests.

### CODE REVIEW

File: tests/mocha/block_test.js [hunk 1]
```diff
@@ -116,29 +116,29 @@ suite('Blocks', function() {
         blocks.B.unplug(true);
         assertUnpluggedHealFailed(blocks);
       });
-      test('A has multiple inputs', function() {
+      test('Parent has multiple inputs', function() {
```
Comment (alschmiedt) [inline tests/mocha/block_test.js:119]: If we are going to change to talk about these as Parent, Middle, Child might be worth it to add a comment above explaining which ones are which.

File: tests/mocha/block_test.js [hunk 2]
```diff
@@ -49,6 +49,29 @@ suite('Blocks', function() {
     delete Blockly.Blocks['statement_block'];
   });
 
+  function createTestBlocks(workspace, isRow) {
```
Comment (alschmiedt) [inline tests/mocha/block_test.js:52]: I like the helper method. If all the tests use this.blocks could we just set that in the parent setup method?

Comment (moniika) [reply tests/mocha/block_test.js:52]: I tried setting this.blocks, but it seems like the "this" in the function is not the same "this" as in the test suites.

Comment (alschmiedt) [reply tests/mocha/block_test.js:52]: I think I meant just having this.block = createTestBlocks() in the parent setup function.

**Pull request merged**